packages/block-events-stream/README.md

```diff
@@ -0,0 +1,34 @@
+# Block events stream
+
+## Example
+
+```ts
+import {
+  createBlockStream,
+  isNonPendingBlock,
+  groupLogsByBlockNumber,
+  blockRangeToLogs,
+} from "@latticexyz/block-events-stream";
+
+const latestBlock$ = await createBlockStream({ publicClient, blockTag: "latest" });
+
+const latestBlockNumber$ = latestBlock$.pipe(
+  filter(isNonPendingBlock),
+  map((block) => block.number)
+);
+
+latestBlockNumber$
+  .pipe(
+    map((latestBlockNumber) => ({ startBlock: 0n, endBlock: latestBlockNumber })),
+    blockRangeToLogs({
+      publicClient,
+      address,
+      events,
+    }),
+    map(({ logs }) => from(groupLogsByBlockNumber(logs))),
+    mergeAll()
```

Comment:
should we add `import { mergeAll } from "rxjs"` for this example?

Reply:
also what's the difference between `map + mergeAll` to `mergeMap`?

Reply:
no real difference! just got here iteratively, can do `mergeMap` instead